config/routes.rb

```diff
@@ -81,5 +81,5 @@
 
   get '/test' => 'static_pages#test'
 
-  root to: redirect('/admin/loans')
+  root to: redirect(path: '/admin/loans', params: { loans: { f: { status_value: ['active'] } } })
```

Comment:
I don't think we can access the url_helpers here. Can you think of a way to reduce the duplication with `admin_loans_path`?